### ISSUE TITLE
Unable to add destination for Datadog source

### ISSUE THREAD

**Issue by acsbendi** [airbytehq/airbyte#22913]
## Environment
- **Airbyte version**:  0.40.32
- **OS Version / Instance**: Ubuntu 22.04 on AWS EC2
- **Deployment**: Docker compose
- **Source Connector and version**: Datadog 0.1.0
- **Destination Connector and version**: Snowflake
- **Step where error happened**: Setup new connection

## Current Behavior
On the Datadog source's page, after clicking the button "add destination" and selecting a destination, Airbyte says "We are fetching the schema of your data source" and starts loading. However, only after a couple of seconds, an error is shown: "Oops! Something went wrong… Unknown error occurred".

## Expected Behavior
The schema of Datadog data source should be loaded correctly.

## Logs
Unfortunately downloading server logs also fails: "Unable to download logs at this time". When inspecting the frontend, the following error is shown in the console:
```
TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at Ul (SchemaError-e083e8fe.js:72:6583)
    at bo (index-01f874f4.js:37:52767)
    at to (index-01f874f4.js:37:62191)
    at ra (index-01f874f4.js:37:61528)
    at di (index-01f874f4.js:37:61122)
    at wa (index-01f874f4.js:41:11784)
    at ya (index-01f874f4.js:41:932)
    at kc (index-01f874f4.js:41:863)
    at cn (index-01f874f4.js:41:722)
```

## Steps to Reproduce
1. Add Datadog source
2. Try adding a destination

## Are you willing to submit a PR?
Yes, but need help with how to fix it.

**Comment by leo-schick** [comment]
I have a similar UI error ("Unknown error occurred") described [here](https://github.com/airbytehq/airbyte/issues/23833#issuecomment-1473944817). But it looks like that something other is wrong at your installation when you even cannot download the server logs.

Can you upgrade to a newer version e.g. 0.42.0 and check if you can download the server logs there?

**Comment by acsbendi** [comment]
This seems to be fixed as of v0.50.5, closing.
